docs/STATE-CONFIGS.md

```diff
@@ -0,0 +1,194 @@
+# Reggie state configs
+
+State configuration files are used to describe the state as well as how we ingest the raw voter data and turn it into more usable data.
+
+## Location
+
+State config files are found in the `reggie/configs/data/` directory and are in YAML format.
+
+## Required fields
+
+The following fields are required in the config for Reggie:
+
+* ??
+
+The following fields are required in the config for VoteShield:
+
+* `voter_status`
+* `birthday_identifier`
+* `party_identifier`
+* ??
```

Comment:
the full set needed for VS is probably close to everything. Let me know if you want help trying to construct these lists